### ISSUE TITLE
Document which network is used on ethers.io

### ISSUE THREAD

**Issue by fulldecent** [ethers-io/ethers.io#18]
When using ethers.io you are apparently connect to some Ethereum network. It is not clear which network you are connected to (Ropsten)? This should be explained on the home page.

Once I know which network it is then I can find a faucet and start using apps.

**Comment by ricmoo** [comment]
Yes, I should add more documentation on this too. As a quick overview:

ethers.io connects to homestead

ropsten.ethers.io connects to ropsten

rinkeby.ethers.io... kovan.ethers.io...

The details of custom-rpc.ethers.io needs more documentation too, which allows ethers to be used for custom networks.

**Comment by fulldecent** [comment]
Thank you for the quick reply. I didn't choose ethers.io immediately for the project I'm working on.

But it looks nice and I will reconsider an alternative to MetaMask in the future.
